Makefile

```
@@ -0,0 +1,13 @@
+generate:
+	tuist generate
+
+clean:
```

checkmake (0.2.2)

[warning] Target body for "clean" exceeds allowed length of 5 (7).

(maxbodylength)